Modules/Registration/Metricsv4/include/itkMattesMutualInformationImageToImageMetricv4.hxx

```diff
@@ -307,22 +307,40 @@ MattesMutualInformationImageToImageMetricv4<TFixedImage,
                                             TInternalComputationValueType,
                                             TMetricTraits>::ComputeResults() const
 {
+  if (this->GetNumberOfValidPoints() == 0)
```

Comment:
> Use alias variable names that more clearly document what the intent
of the workspace ivars that are re-used memory this stage of the processing.

Unclear sentence in the commit message.